reconcile/utils/terrascript_client.py

```diff
@@ -2682,11 +2682,19 @@ def get_tf_iam_access_key(self, user_tf_resource,
         # outputs
         # aws_access_key_id
         output_name_0_13 = output_prefix + '__aws_access_key_id'
-        output_value = '${' + tf_resource.id + '}'
-        tf_resources.append(Output(output_name_0_13, value=output_value))
+        output_value_id = '${' + tf_resource.id + '}'
```

Comment:
f-strings are more readable...
```python
output_value_id = f"${{tf_resource.id}}'
```

Reply:
i'd say this is out of scope? there a lot of these in our code base

Reply:
Fine by me, I've approved the changes nevertheless.